tests/app/celery/test_scheduled_tasks.py

```diff
@@ -700,3 +701,16 @@ def test_auto_expire_broadcast_messages(
         )
     else:
         assert not mock_celery.called
+
+
+def test_remove_yesterdays_planned_tests_on_govuk_alerts(
```

Comment:
should we have an end-to-end test, with freezetime to show that alert is created on one day, then next day it is not shown in planned alerts anymore?

Reply:
I don’t know how you can freeze time like that on a functional, end-to-end test.

I did look to see if there was at least tests for some of the other scheduled task timings but couldn’t see anything to copy.

Reply:
Ahh, maybe it's too much work to establish a totally new pattern then, it's ok 👍🏼

Reply:
I thought of an end-to-end unit test like this:

```
freeze time on 11.11.2021
create the alert on that day
freeze time on 12.11.2021
run remove_yesterdays_planned_tests_on_govuk_alerts()
assert the alert is treated differently